gallery/plot_visualization_utils.py

```diff
@@ -65,28 +66,23 @@ def show(imgs):
 
 #####################################
 # Naturally, we can also plot bounding boxes produced by torchvision detection
-# models.  Here is demo with a Faster R-CNN model loaded from
+# models.  Here is a demo with a Faster R-CNN model loaded from
 # :func:`~torchvision.models.detection.fasterrcnn_resnet50_fpn`
-# model. You can also try using a RetinaNet with
-# :func:`~torchvision.models.detection.retinanet_resnet50_fpn`, an SSDlite with
-# :func:`~torchvision.models.detection.ssdlite320_mobilenet_v3_large` or an SSD with
-# :func:`~torchvision.models.detection.ssd300_vgg16`. For more details
```

Comment:
I'm not sure why we kept adding more and more models here but I ended up removing them all. I think people get it that they can use other models too, without listing them all.

Reply:
It was for the backlinks. See https://pytorch.org/vision/main/models/generated/torchvision.models.detection.retinanet_resnet50_fpn.html#torchvision.models.detection.retinanet_resnet50_fpn at the bottom, theres a "Examples using retinanet_resnet50_fpn:" section.

Not mentioning the model builders in the example removes that. I don't have a strong opinion, I agree it's not really scalable if we have to remember to link all models from the examples. But it's also nice to have the backlinks... IDK. Maybe we can do that more manually.

Reply:
Let's add manually, maybe with refs from the documentation pages of the methods as you recommended previously offline. The specific list was not complete and missing several new models.